cpp/oneapi/dal/graph/detail/undirected_adjacency_vector_graph_impl.cpp

```diff
@@ -14,13 +14,11 @@
 * limitations under the License.
 *******************************************************************************/
 
-#include "oneapi/dal/graph/detail/undirected_adjacency_array_graph_impl.hpp"
+#include "oneapi/dal/graph/detail/undirected_adjacency_vector_graph_impl.hpp"
 
 namespace oneapi::dal::preview::detail {
 
-template class ONEDAL_EXPORT undirected_adjacency_array_graph_impl<empty_value,
-                                                                   empty_value,
-                                                                   empty_value,
-                                                                   std::int32_t,
-                                                                   std::allocator<char>>;
+template class ONEDAL_EXPORT topology<int32_t>;
+
+template class ONEDAL_EXPORT dal::v1::array<float>;
```

Comment:
1) is it required to instantiate `array<float>`?
2) remove v1

Reply:
1. Instantiation is required to be sure the optimized topology is fully in the lib.